karma.conf.js

```diff
@@ -6,19 +6,13 @@ module.exports = function (config) {
 
         // list of files / patterns to load in the browser
         files: [
-            'source/**/*.js',
-            'test/**/*.js',
+            'source/**/*.ts',
+            'test/**/*.ts',
         ],
 
         preprocessors: {
-            'source/**/*.js': ['browserify'],
-            'test/**/*.js': ['browserify'],
-        },
-        browserify: {
```

Comment:
I'm not sure if there is anything still requiring this step.

Reply:
It should be needed especially in this Karma related configuration.

Reply:
@radibit ok, thanks!